### ISSUE TITLE
About Performance on Imagenetdogs 

### ISSUE THREAD

**Issue by kasteric** [Hzzone/ProPos#10]
Hi, I want to ask a small question, the performance I implemented on Imagenetdogs using BYOL/resnet34 only reaches 56% ACC, far lower than the reported 69% ACC, is there anything I miss :)

**Comment by kasteric** [comment]
Here is my reproduced results, with 56.4% ACC,
![image](https://user-images.githubusercontent.com/132050661/236749533-7fee4005-fcc1-47a3-97b4-e2ba30645a34.png)
 and here is my config file:
```
batch_size: 128
num_devices: 4
momentum_base: 0.996
momentum_max: 1.0
momentum_increase: true
dataset: imagenetdogs
eval_metric:
  - nmi
  - acc
  - ari
whole_dataset: true
encoder_name: resnet34
epochs: 1000
feat_dim: 256
hidden_size: 4096
#img_size: 96
img_size: 96
lambda_predictor_lr: 10
learning_rate: 0.05
learning_eta_min: 0.
reassign: 1
save_freq: 50
save_checkpoints: true
shuffling_bn: true
symmetric: true
temperature: 0.5
#use_gaussian_blur: false
use_gaussian_blur: true
warmup_epochs: 50
weight_decay: 0.0005
dist: true
data_resample: true
v2: true
#byol_transform: true
test_resized_crop: true
use_copy: true

model_name: propos
cluster_loss_weight: 0.
latent_std: 0.
```
Could u please give me more prompts on how to train ImageNetdogs? Thanks a lot!

**Comment by Hzzone** [comment]
It seems that you have changed the batch size. Please refer to this config https://github.com/Hzzone/ProPos/blob/cc35d8ca276aab8aa5dfb638fa15f919cf483824/config/imagenetdogs_r34_propos.yml

**Comment by kasteric** [comment]
> It seems that you have changed the batch size. Please refer to this config https://github.com/Hzzone/ProPos/blob/cc35d8ca276aab8aa5dfb638fa15f919cf483824/config/imagenetdogs_r34_propos.yml

Thanks! What about the cosine learning rate scheduler, should I keep the same with CIFAR10?

**Comment by Hzzone** [comment]
No.
